tests/integration/marketplacesuite.nim

```diff
@@ -45,25 +45,40 @@ template marketplacesuite*(name: string, body: untyped) =
     proc periods(p: int): uint64 =
       p.uint64 * period
 
-    proc createAvailabilities(datasetSize: int, duration: uint64) =
+    # Purposely overshooting
+    # TODO: find a better way to compute slot size for the given
+    # dataset size
+    # I found this snippet in tests/integration/testproofs.nim:
+    # let datasetSizeInBlocks = 3
+    # let data = await RandomChunker.example(blocks=datasetSizeInBlocks)
+    # # original data = 3 blocks so slot size will be 4 blocks
+    # let slotSize = (DefaultBlockSize * 4.NBytes).Natural.u256
+    proc availabilityTotalSizeForDataSize(dataSize: int): UInt256 =
+      (dataSize * 2).u256
```

Comment:
I think it would end up being dependent upon the number of blocks in the dataset, and the EC params of the storage request (nodes + tolerance) for the individual test, so those parameters should be included, eg
```nim
proc datasetSize(blocks, nodes, tolerance: int): UInt256 =
  (DefaultBlockSize * (nodes + tolerance).NBytes).Natural.u256
```

Reply:
I have added two helper functions to `marketplacesuite.nim`:

```nim
proc slotSize(blocks: int): UInt256 =
  (DefaultBlockSize * blocks.NBytes).Natural.u256

proc datasetSize(blocks, nodes, tolerance: int): UInt256 =
  (nodes + tolerance).u256 * slotSize(blocks)
```

They seem to be giving me correct values.